scripts/scil_summarize_bundles_similarity_measures.py

```diff
@@ -319,7 +319,7 @@ def main():
                     measure_dict[measure_name])
 
     with open(args.out_json, 'w') as outfile:
-        json.dump(output_measures_dict, outfile)
+        json.dump(output_measures_dict, outfile, indent=1)
```

Comment:
Use add_json_args and use it here.

Reply:
C'est actually un erreur, je vais l'enlever